src/mlpack/methods/isomap/mds.hpp

```diff
@@ -0,0 +1,121 @@
+/**
+ * @file mds.hpp
```

Comment:
Should we put this in its own `mds/` directory and create a binding for that also?

Reply:
Yes, I think that is better. I will do that.